graphstorm-processing/graphstorm_processing/config/label_config_base.py

```diff
@@ -0,0 +1,112 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License").
+You may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import abc
+from typing import Any, Dict, List, Optional, Sequence
+
+
+class LabelConfig(abc.ABC):
+    """Basic class for label config"""
+
+    def __init__(self, config_dict: Dict[str, Any]):
+        self._config = config_dict
+
+        cols = config_dict["column"]
```

Comment:
As https://github.com/awslabs/graphstorm/blob/main/python/graphstorm/gconstruct/transform.py#L1324.
For a link prediction task, label config may not have col_name and label_name.

Reply:
Right, I'll check our implementation for the converter and make sure we remove this assumption.